src/NzbDrone.Core/Datastore/Migration/209_postgres_update_timestamp_columns_to_with_timezone.cs

```diff
@@ -0,0 +1,62 @@
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"ImportListStatus\" ALTER COLUMN \"MostRecentFailure\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"ImportListStatus\" ALTER COLUMN \"DisabledTill\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"IndexerStatus\" ALTER COLUMN \"InitialFailure\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"IndexerStatus\" ALTER COLUMN \"MostRecentFailure\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"IndexerStatus\" ALTER COLUMN \"DisabledTill\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"IndexerStatus\" ALTER COLUMN \"CookiesExpirationDate\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"MetadataFiles\" ALTER COLUMN \"LastUpdated\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"MetadataFiles\" ALTER COLUMN \"Added\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"MovieFiles\" ALTER COLUMN \"DateAdded\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"Movies\" ALTER COLUMN \"LastInfoSync\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"Movies\" ALTER COLUMN \"LastDiskSync\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"Movies\" ALTER COLUMN \"InCinemas\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"Movies\" ALTER COLUMN \"Added\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"ImportListMovies\" ALTER COLUMN \"PhysicalRelease\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"ImportListMovies\" ALTER COLUMN \"DigitalRelease\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"PendingReleases\" ALTER COLUMN \"Added\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"ScheduledTasks\" ALTER COLUMN \"LastExecution\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"ScheduledTasks\" ALTER COLUMN \"LastStartTime\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"SubtitleFiles\" ALTER COLUMN \"Added\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"SubtitleFiles\" ALTER COLUMN \"LastUpdated\" TYPE timestamp with time zone");
+            IfDatabase("postgres").Execute.Sql("ALTER TABLE \"VersionInfo\" ALTER COLUMN \"AppliedOn\" TYPE timestamp with time zone");
```

Comment:
This resolves the fact that PG suggest data types should not be of type "timestamp without time zone"; however Fluent cannot natively set a DateTime column as "with timezone" - @Qstick / @ta264  do we want to go down this route as it will need to be done whenever a new column is being added to PG. I personally don't think we should, but wanted some views on it

Reply:
Yea, I'm not keen to go this route unless we have to. If the other bit fixes the issue and new columns are not going to be a problem then I say we just stick with that.